### PR TITLE
Added call to action for donations to foundation page

### PR DESCRIPTION
**Pull request by JackStouffer**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -627,6 +627,20 @@ a.btn:hover
     color: #333;
 }
 
+.btn.donate-large
+{
+    font-size: 22px;
+    padding: 0.4em 1em;
+    background-color: #D8645E;
```
Comment (wilzbach) [inline css/style.css]: Why not `#B03931`?

Yours:

image

---

image

File: css/style.css [hunk 2]
```diff
@@ -627,6 +627,20 @@ a.btn:hover
...
+    color: white;
+    border: #D8645E;
+}
+.btn.donate-large:hover
+{
+    background-color: #F47069;
```
Comment (wilzbach) [inline css/style.css]: Why not `#98312A` (similar to the menu)?

**Comment by JackStouffer** [comment]
No particular reason. I saw it in the CSS and thought it looked better. I can change it if you want.

**Comment by wilzbach** [comment]
> No particular reason. I saw it in the CSS and thought it looked better. I can change it if you want.

Changed it myself as a separate commit (so it's easy to revert).
The CSS header says:

```
Reds:

#B03931 - our main red
#742620 - focus/hover

#D8645E - on dark background
#F47069 - focus/hover on dark background
```

So I would stick to the "main red".

**Pull request merged**